TaskModules/powershell/Sanitizer/Strings/resources.resjson/de-DE/resources.resjson

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Why do we need empty resource files?

Reply:
It's for the localization team.
They should check for the new strings and add translations into .resjson files.
I'm now sure that they can create new files, so I just created empty files, for expected languages.

Reply:
Since they have permissions to modify files they should be able to create them as well. Let's remove empty files since those are just a noise now.

Reply:
Done👌